commit a952052c114a0a0f9486a347d5145c34ae8288af
Author: bertin <macportsuser@macports.org>
Date:   Fri Oct 10 19:10:40 2025 +0200

    /opt/local/linux-ports/audio/audacity/files/patch-use-MPlame.diff

diff --git a/modules/import-export/mod-mp3/ExportMP3.h b/modules/import-export/mod-mp3/ExportMP3.h
index 1d15c36d7..35ff4c2a0 100644
--- a/modules/import-export/mod-mp3/ExportMP3.h
+++ b/modules/import-export/mod-mp3/ExportMP3.h
@@ -22,7 +22,7 @@ enum MP3RateMode : unsigned {
    MODE_CBR,
 };
 
-#if defined(__WXMSW__) || defined(__WXMAC__)
+#if defined(__WXMSW__)
 #define MP3_EXPORT_BUILT_IN 1
 #endif
 
